<?xml version="1.0"?>
<launch>
    <arg name="mav_name" default="firefly"/>

    <include file="$(find iarc_simulation_tools)/launch/default.launch">
        <arg name="mav_name" value="$(arg mav_name)"/>
    </include>

    <group ns="$(arg mav_name)">
        <node pkg="led_detector" type="led_detector_node" name="led_detector_node" output="screen">
            <rosparam file="$(find led_detector)/config/param.yaml"/>
            <remap from="image_raw" to="camera_rgb/image_raw"/>
        </node>
        <node pkg="pose_estimator" type="pose_estimator_node" name="led_pose_estimator_node" output="screen">
            <rosparam file="$(find pose_estimator)/config/param.yaml"/>
            <remap from="odom" to="ground_truth/odometry"/>
            <remap from="centre_coord" to="led_detector_node/centre_coord"/>
        </node>
    </group>

</launch>
